docs/content/component-data.md

```diff
@@ -9,56 +9,33 @@ SpatialOS components are generated from `.schema` files into components that the
 ### Overview
 
 A `struct`, which implements `Unity.Entities.IComponentData` and `Improbable.Gdk.Core.ISpatialComponentData`,
-is generated for each [blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) SpatialOS component.
-A `class`, which extends `UnityEngine.Component` and implements `Improbable.Gdk.Core.ISpatialComponentData` is generated for each non-blittable SpatialOS component.
+is generated for each SpatialOS component.
 
 Each of these structs will be named `SpatialOS[schemalang component name]` and will contain only the schema data fields.
 It will **Not** contain any fields or methods pertaining to [commands](commands.md) or [events](events.md) defined on that component. 
 
-For example, the following component contains only blittable fields, so an `IComponentData` will be generated, similar to the example below.
+For example:
 
 Schemalang
 ```
-component Blittable {
+component Example {
   id = 1;
   int32 value = 1;
 }
 ```
 Generated component
 ```	csharp
-struct SpatialOSBlittable : IComponentData, ISpatialComponentData
+public struct SpatialOSExample : IComponentData, ISpatialComponentData
 {
   public int Value;
 }
 ```
 
-However the following component contains non-blittable fields, so a `Component` will be generated, similar to the example below.
-
-Schemalang
-```
-component NonBlittableComponent {
-  id = 2;
-  int32 int_blittable = 1;
-  map<int32, float> map_non_blittable = 2;
-  string string_non_blittable = 3;
-}
-```
-Generated component
-```	csharp
-class SpatialOSNonBlittable : Component, ISpatialComponentData
-{
-  public int IntBlittable;
-  public Dictionary<int, float> MapNonBlittable;
-  public string StringNonBlittable;
-}
-```
-
 ### Reading and writing
 
 When a SpatialOS entity is [checked out](entity-checkout-process.md), the generated components will be automatically added to the corresponding ECS entity. 
 
 A component's value can be read by injecting that component into a system, just like any other ECS component.
```

Comment:
Does this mean the generated components mentioned in the paragraph above? It's not clear if that is what it means.
Suggest:
"These generated component values can be read by injecting ......"